examples/OmeroClients/collectionmethods.py

```diff
@@ -8,14 +8,14 @@
 EventI = omero.model.EventI
 PixelsI = omero.model.PixelsI
 
-image = ImageI(long(1), True)
-image.getDetails().setUpdateEvent(EventI(1L, False))
+image = ImageI(int(1), True)
```

Comment:
No objection, but the `int` here is almost certainly no longer needed.

Reply:
removed